packages/viz/src/lib/sources/CARTOSource.ts

```diff
@@ -20,7 +22,7 @@ function getSourceType(source: string) {
   return containsSpace ? 'sql' : 'dataset';
 }
 
-interface CARTOLayerProps extends LayerProps {
+interface CARTOLayerProps extends SourceProps {
```

Comment:
what if we rename it to `CARTOSourceProps`?

Reply:
Done

Reply:
Done